tests/osaka/eip7692_eof_v1/eip4200_relative_jumps/test_rjump.py

```diff
@@ -116,7 +116,7 @@ def test_rjump_truncated_rjump_2(
 ):
     """EOF1I4200_0002 (Invalid) EOF code containing truncated RJUMP."""
     eof_test(
-        data=Container.Code(Op.RJUMP + Op.STOP),
+        data=Container.Code(Op.RJUMP + b"\x00"),
```

Comment:
to address @marioevz 's concerns, maybe this should be `Op.RJUMP + verbatim(b"\x00")`, and `verbatim` produces some type which can be added to `Bytecode` but isn't `bytes` explicitly, possibly `Bytecode` would be fine? This would make spotting abuse much easier.

Basically `verbatim` could be what @chfast envisioned for the constructor `Bytecode`.